app/src/main/java/org/mozilla/fenix/components/metrics/MetricsUtils.kt

```diff
@@ -28,7 +28,7 @@ object MetricsUtils {
      * Possible sources for a performed search.
      */
     enum class Source {
-        ACTION, SHORTCUT, SUGGESTION, TOPSITE, WIDGET, NONE
+        ACTION, ASSIST, SHORTCUT, SUGGESTION, TOPSITE, WIDGET, NONE
```

Comment:
Drive-by thought - Since this revolves around search, I wanted to make sure we're mindful about collecting telemetry as well. I do remember doing something like this to ensure we have the right telemetry collected. Without looking too deeply into how the metrics are collected, I am wondering if you had a chance to test the telemetry collected here and if this was the only change we needed for telemetry?

Reply:
We got a wiki for testing telemetry at https://github.com/mozilla-mobile/fenix/wiki/Test-telemetry-pings

Reply:
Since this is a new feature - we should file a separate issue for collecting telemetry instead of feature-creeping this one.

That said, it should be a fast-followup to your points as well about search.

Reply:
I tested the telemetry and it worked, but I agree let's follow up telemetry with a separate issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1808043